ibis/backends/conftest.py

```diff
@@ -288,7 +289,20 @@ def pytest_collection_modifyitems(session, config, items):
     except ImportError:
         pyspark = None
 
+    unrecognized_backends = set()
     for item in items:
+        # Yell loudly if unrecognized backend in notimpl, notyet or never
+        for name in ("notimpl", "notyet", "never"):
+            for mark in item.iter_markers(name=name):
+                if backend := set(util.promote_list(mark.args[0])).difference(
+                    ALL_BACKENDS
+                ):
+                    unrecognized_backends.add(
+                        f"""Unrecognize backend(s) [{backend}] passed to {name} marker in
```

Comment:
```suggestion
                        f"""Unrecognize backend(s) {backend} passed to {name} marker in
```

Probably don't need the extra `[]` since `backend` is a `set`!

Reply:
good call!